src/common/components/hosts/HostStatus.tsx

```diff
@@ -193,7 +193,11 @@ type WithHostStatusPopoverProps = AdditionNtpSourcePropsType &
 const WithHostStatusPopover: React.FC<WithHostStatusPopoverProps> = (props) => (
   <Popover
     headerContent={<div>{props.title}</div>}
-    bodyContent={<HostStatusPopoverContent {...props} />}
+    bodyContent={
+      <div style={{ maxHeight: '25rem', overflow: 'auto' }}>
```

Comment:
It's hard to say if an absolute value is going to work in all situations.
Could it be more adaptable if we used a size related to the viewport height?

Reply:
Changed to `33vh`.